packages/Button/src/Button.js

```diff
@@ -146,9 +146,24 @@ const Button = React.forwardRef((props, ref) => {
 
   const bestTabIndex = isButtonDisabled && tabIndex === null ? -1 : tabIndex || 0;
 
+  function getClassName() {
```

Comment:
Instead of tacking on a custom className, can we keep this component agnostic of classes and use a custom data attribute instead? `data-show-focus`?

Reply:
k but i'd like to keep the words `has` and `forced` in there: `data-has-forced-focus`